packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -27,11 +39,13 @@ function Export({
 }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const onAction = useContext(ActionContext);
+  const { show } = useToast();
   const [isBusy, setIsBusy] = useState(false);
   const [pass, setPass] = useState('');
   const [error, setError] = useState('');
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
 
-  const _goHome = useCallback(() => onAction('/'), [onAction]);
+  const _goTo = useCallback((path: string) => () => onAction(path), [onAction]);
```

Comment:
I know we discussed this already, but I think you mentioned that the _`useCallback` by default_ pattern is enforced by eslint - but I think it isn't, is it? I removed it locally, run lint and it did not throw an error.

I'm returning to this topic, because while in other places it makes little sense, in this place it makes literally zero sense :P That's because `useCallback` memoizes the external function only and since `_goTo` is always used by being invoked in the same component / render function that it's declared in and it returns another function, that another function is not memoized on its own and thus creates that _new-reference-caused-rerendering_ problem anyway.

Reply:
Use callback per se is not enforced - the no-bind rule is. But their code style is like that - each file written by polkadot contributors was using the `useCallback` for every method in components. I find it a bit confusing if parts of the app will be written with useCallback on every small fn and parts not. I'm a big supporter of one code style even if I'm not fully agree with it. I'm cool with one big refactor task to make all of them uniform if not using useCallback but that would require extensive tests for which we don't have time now :/

Reply:
Gotcha 👍